ext/hal/nxp/mcux/CMakeLists.txt

```diff
@@ -30,3 +35,7 @@ add_subdirectory_ifdef(
   CONFIG_IEEE802154_KW41Z
   middleware/wireless/framework_5.3.3
   )
+
+enable_language(C ASM)
+
+zephyr_sources_ifdef(CONFIG_SOC_LPC54114 devices/${MCUX_DEVICE}/amp_startup.S)
```

Comment:
Any reason not to keep the same path & filename as the SDK?

Reply:
Fixed, kept the same name.